doc/changes/latest.inc

```diff
@@ -46,6 +46,8 @@ Enhancements
 
 - Add ``picks`` parameter to :func:`mne.preprocessing.fix_stim_artifact` to specify which channel needs to be fixed, by `Alex Gramfort`_ (:gh:`8482`)
 
+- Add :class:`mne.decoding.SSD`, implementation of the spatio-spectral-decomposition (SSD) method, by `Denis Engemann`_ and `Victoria Peterson`_ (:gh:`7070`)
```

Comment:
Is there a concrete example somewhere @larsoner -- the new doc is sweet but looking at a concrete example would be helpful.

Should it be

```
- Add :class:`mne.decoding.SSD`, implementing spatial filtering with spatio-spectral-decomposition (SSD) (:gh:`7070` by `Denis Engemann`_ and **new contributor** `Victoria Peterson`_)
```

Reply:
Yes, I've found on the bug area!

it would be like 

```
- .. |Victoria Peterson| replace:: **Victoria Peterson**

- Add :class:`mne.decoding.SSD` for spatial filtering with spatio-spectral-decomposition (SSD), by `Denis Engemann`_ and **by new contributor** |Victoria Peterson|_ (:gh:`7070`)
```

Reply:
I'll let @larsoner confirm

Reply:
```
- .. |Victoria Peterson| replace:: **Victoria Peterson**

- Add :class:`mne.decoding.SSD` for spatial filtering with spatio-spectral-decomposition (SSD), (:gh:`7070` **by new contributor** |Victoria Peterson|_ and `Denis Engemann`_)
```

Reply:
alright ! thanks ! so I'll commit that @dengemann